docs/install/kubernetes/README.md

```diff
@@ -210,5 +210,5 @@ forge:
 ## Upgrade
 
 - Run `helm repo update flowforge` to pull the latest version
-- Check the [README.md](https://github.com/flowforge/helm/blob/main/helm/flowforge/README.md) for any new options to configure in `customization.yml`
+- Check the [README.md](https://github.com/FlowFuse/helm/blob/main/helm/flowforge/README.md) for any new options to configure in `customization.yml`
```

Comment:
I think we need a comment here about the change of the repo URL before the bullet in the list

Reply:
@hardillb please can you PR a doc update for this for tomorrow. I need this in to unblock the build/